[2.0] FAQ: update fees for coinjoin (#1128)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

:::

:::details
### What are the fees for the coinjoin?

> Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a [coordinator](https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi.Backend) fee + [mining fees](https://mempool.space/).

Reviewer: I don't see a reason to link to those:

```suggestion
Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a coordinator fee + mining fees.
```

> Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a coordinator fee + mining fees.
> Inputs of 0.01 BTC or below don't pay coordinator fees.
> And remixes, even after one transaction, don't pay coordinator fees. 
> Thus, a payment made with coinjoined funds allows the sender and the recipient to remix their coins without paying any coordinator fees.

Reviewer: How about this tweak:

```suggestion
Coinjoin fees are composed of coordination fees and mining (network) fees.
Coins with a value above 0.01 BTC pay 0.3% as a coordination fee + mining fees.
Coins of 0.01 BTC or below don't pay coordination fees.
Remixes, even after one transaction, also don't pay coordination fees. 
Thus, a payment made with coinjoined funds allows the sender and the recipient to remix their coins without paying any coordination fees.
```

Author: good.
imo `UTXO` is better fit in this context instead of `coins`?

Reviewer: I think coins is easier to understand for the average person.
I have updated the suggestion to say `Coins (UTXOs)`.
